pkg/utils/authenticationUtils.go

```diff
@@ -67,3 +69,37 @@ func readJWTFromFS(tokenFilePath string) (string, error) {
 	}
 	return string(token), nil
 }
+
+// RegionFromImage parses region from image url
+func RegionFromImage(image string) (string, error) {
```

Comment:
We've added a new cloud specific utils folder. Maybe you could add an awsutils folder like azure has done here? https://github.com/deislabs/ratify/tree/main/pkg/utils